config/exercise_readme.go.tmpl

```diff
@@ -17,11 +17,28 @@ Execute the tests with:
 $ cargo test
 ```
 
-All but the first test have been ignored.  After you get the first test to
-pass, remove the ignore flag (`#[ignore]`) from the next test and get the tests
-to pass again.  The test file is located in the `tests` directory.   You can
-also remove the ignore flag from all the tests to get them to run all at once
-if you wish.
+All but the first test have been ignored. If you wish to run every possible test use:
+
+```bash
+$ cargo test -- --ignored
+```
+
+To run specific test, for example `some_test`, you can use:
+
+```bash
+$ cargo test some_test
+```
+
+To find test names, view the tests source file wich is located in `tests` directory.
+Every function that contains `#[test]` flag above it's name is a separate test.
+If the specfic test is ignored use:
+
+```bash
+$ cargo test some_test -- --ignored
+```
+
+Alternatively you can edit the tests source file and remove the `#[ignore]` flag
```

Comment:
This should remain the primary encouraged method of proceeding through the exercise. It's fine to list some alternatives, but iterative TDD is an Exercism standard.

Reply:
Another possible method you might suggest might be to use a script:

```sh
for test in tests/*.rs; do
   sed -i -e '/#\[ignore\]/{
      s/#\[ignore\]\s*//
      /^\s*$/d
   }' $test
done
```

This addition is not required.